starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2941,13 +2987,13 @@ sub set_ccd_temps {
         if ($self->{ccd_temp} > $config{ccd_temp_yellow_limit}) {
             push @{ $self->{fyi} },
               sprintf("Effective guide temperature %.1f C\n",
```

Comment:
I assume you are leaving the `Effective` here for API consistency?

Reply:
Nope, just missed this in my search.  I'll update this and double-check the parser.

Reply:
Actually this doesn't even hit the parser because these are just in the "warnings" section.  This whole block could go so I cut it.